cloud/migrations/005_lessons_unique_constraint.sql

```
@@ -0,0 +1,29 @@
+
+WITH ranked AS (
+    SELECT
+        id,
+        ROW_NUMBER() OVER (
+            PARTITION BY brain_id, description
+            ORDER BY updated_at DESC NULLS LAST, confidence DESC, id DESC
+        ) AS rn
+    FROM lessons
+)
+DELETE FROM lessons
+WHERE id IN (SELECT id FROM ranked WHERE rn > 1);
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)